src/applications/vaos/tests/mocks/helpers.js

```diff
@@ -1,3 +1,5 @@
+/** @module testing/mocks/helpers */
```

Comment:
Not finished with all the functions in this file, but the PR is already really big. Will make a second one.